### ISSUE TITLE
https://dweb.blahblah.com/ is redirected to https://dweb.me/arc/archive.org/details

### ISSUE THREAD

**Issue by abhidas17695** [abhidas17695/dweb-ext#22]
Steps to reproduce this bug :
1. Visit dweb.archive.org. 
2. Wait for the re-direction to take place
3. Open a new tab and visit https://dweb.blahblah.com/
4. The new URL loaded is https://dweb.me/arc/archive.org/details?
Whatever the last URL redirected to , this URL redirects to the same

**Comment by mitra42** [comment]
So problem could be in my code or your code, next step would be to generate some console logging - when I follow these steps above all I see is 
```
Navigated to chrome-error://chromewebdata/
2Navigated to https://dweb.me/archive/archive.html
```
We need some console logging which checks 

1. what you send at the boundary between your code and mine, and then
2. what you get back ...

That way we should be able to quickly (and repeatedly) determine whether the problem is in your code pre-call; my code; or code-post-call

**Comment by abhidas17695** [comment]
I will add logging for that

**Comment by abhidas17695** [comment]
I've added the logging at the boundaries.
You should see **URL intercepted is https://dweb.blahblah.com/** and **URL to load is https://dweb.me/archive/archive.html**

**Comment by mitra42** [comment]
I think its caused by #23, and using the result of previous request, which has been stored in bad (global) location.  Lets fix #23 and recheck this.

**Comment by mitra42** [comment]
Works now - fixed by #23
